### PR TITLE
Beta Release 0.3.0

### LINKED ISSUE #131: Philippine passports are not validated correctly
**Describe the bug**
You have to add a number to the end of a correct Philippine passport number to be considered “correct” by the validator.

**To Reproduce**

1. Log in as an account that has not passed the Welcome process
2. Click on “Continue”
3. Enter a valid Philippine passport number
4. See error (until one digit added to the end)

**Expected behavior**
A correct Philippine passport number should be considered valid.

**Screenshots**
image

**Additional context**
The correct RegExp is:
```ts
/^[A-Za-z][0-9]{7}[A-Za-z]$/
```

### LINKED ISSUE #132: Long teacher name overflows Schedule Period
**Describe the bug**
Some teacher names are too long to fit in a Schedule Period and split into 2 lines, the 2nd of which is outside the Period.

**To Reproduce**
Konjanart (Thai teacher)’s name in Thai is an example.

1. Log in as a student
2. Go to `/learn/504`
3. Find “โกญจนาท”
4. See the name split into 2 lines

**Expected behavior**
The name should stay in 1 line. If it is longer than the Schedule Period, it should either just keep going (and clip the overflow) or truncate.

**Screenshots**
image

### PR DESCRIPTION
**Pull request by SiravitPhokeed**
image
image
image

**Features**

- School Orders page (with search and pagination)
- Documents page (with search and pagination)
- Replace News Card with News List Item
- Links are shortened
- News Banner is now consistently 16:9 everywhere
- Replace About page with a footnote in Landing
- Description `<meta>` tag
- Includes component

**Fixes**

- Long links break Article pages
- Overlapping periods are mislabeled as “Elective period” for teachers
- A teacher can teach many Subjects in a Period (Periods can overlap from the teacher side)
- Schedule Period displays all teachers teaching that subject, not all teachers actually teaching that period
- Some teachers miss the “Period duration” text field in Edit Period and create 2 individual periods to represent a longer one
- Some passports are not validated correctly
  Fixes #131
- Long teacher name split into many lines in Schedule Period
  Fixes #132

### CODE REVIEW

**Review by Jimmy-Tempest — APPROVED** [review]
LGTM!!!!

**Comment by Temp9699** [comment]
LGTM!!!

File: public/static/locales/en-US/schedule.json [hunk 1]
```diff
@@ -11,6 +11,7 @@
         "close": "Close"
       }
     },
+    "overlap": "Many subjects",
```
Comment (ImSadudee) [inline public/static/locales/en-US/schedule.json:14]: Consider changing to "Multiple subjects" instead.

**Review by ImSadudee — APPROVED** [review]
LGTM!

File: src/utils/backend/news/document.ts [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+
+// Types
+import { BackendDataReturn } from "@utils/types/common";
+import {
+  NewSchoolDocumentCount,
+  SchoolDocument,
+  SchoolDocumentType,
+} from "@utils/types/news";
+import { supabase } from "@utils/supabase-client";
+import { db2SchoolDocument } from "../database";
```
Comment (SiravitPhokeed) [inline src/utils/backend/news/document.ts:12]: Imports should be grouped.

**Review by SiravitPhokeed — COMMENTED** [review]
LGTM!

**Review by Temp9699 — APPROVED** [review]
LGTM!!! sorry I forgot

**Pull request merged**